### ISSUE TITLE
Better serving solution?

### ISSUE THREAD

**Issue by radiovisual** [radiovisual/birdwatch#9]
The serving solution changed in the `1.0.0` release. Now, the cache waits for the Birdwatch process to complete (via `async/await`) and then saves the filtered data to a cache file, but only uses the in-memory cache (`this.tweets`) to serve data. I decided on this for the main reason of, the cache file was never getting hit, ever, and if I used it as a "backup" to serve the data before the in-memory cache was ready, then the unit tests (and API) could fail because the cached data might not match up with the data the user cares about. For example, the cache-file might have some `TWEET A`, but the new instance of Birdwatch might say: "only serve me `TWEET B`", but if the in-memory cache (with `TWEET B`) is not ready, it would have served `TWEET A`, just to have something to serve...and that didn't feel right to me. 

That being said, **I am open for discussion**.

**Comment by radiovisual** [comment]
[Cerebral](https://github.com/christianalfoni/cerebral) might be a good fit for populating the cache when the app state changes. Make a branch and test it out.

**Comment by radiovisual** [comment]
The serving solution changed in the `1.0.0` release. Now, the cache waits for the Birdwatch process to complete (via `async/await`) and then saves the filtered data to a cache file, but only uses the in-memory cache (`this.tweets`) to serve data. I decided on this for the main reason of, the cache file was never getting hit, ever, and if I used it as a "backup" to serve the data before the in-memory cache was ready, then the unit tests (and API) could fail because the cached data might not match up with the data the user cares about. For example, the cache-file might have some `TWEET A`, but the new instance of Birdwatch might say: "only serve me `TWEET B`, but if the in-memory cache (with `TWEET B`) is not ready, it would have served `TWEET A`, just to have something to serve...and that didn't feel right to me. 

That being said, **I am open for discussion**.
